feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio_ubus.c

```diff
@@ -50,6 +50,35 @@ int hapd_rrm_set_neighbors(char *name, struct rrm_neighbor *neigh, int count)
 	return ubus_invoke(ubus, id, "rrm_nr_set", b.head, NULL, NULL, 1000);
 }
 
+int ubus_set_channel_switch(const char *if_name, uint32_t frequency,
+			    const char *hw_mode, int channel_bandwidth,
+			    int sec_chan_offset)
+{
+	uint32_t id;
+	char path[64];
+
+	snprintf(path, sizeof(path), "hostapd.%s", if_name);
+
+	if (ubus_lookup_id(ubus, path, &id))
+		return -1;
+	blob_buf_init(&b, 0);
+
+	if (!strncmp(hw_mode, "11n", strlen("11n"))) {
+		blobmsg_add_u8(&b, "ht", 1);
+	} else if (!strncmp(hw_mode, "11ac", strlen("11ac"))) {
+		blobmsg_add_u8(&b, "vht", 1);
+	}
```

Comment:
Don't we need to add something like:

else if (!strncmp(hw_mode, "11ax", strlen("11ax"))) {
 		blobmsg_add_u8(&b, "he", 1);
 	}